inference-engine/tests/functional/plugin/shared/src/low_precision_transformations/concat_neighboring_graph_transformation.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <memory>
+#include <tuple>
+#include <vector>
+#include <string>
+
+#include <ie_core.hpp>
+
+#include "common_test_utils/common_utils.hpp"
+#include "functional_test_utils/plugin_cache.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "functional_test_utils/blob_utils.hpp"
+
+#include "ngraph_functions/pass/convert_prc.hpp"
+
+#include "low_precision_transformations/concat_neighboring_graph_transformation.hpp"
+
+
+namespace LayerTestsDefinitions {
+
+std::string ConcatNeighboringGraphTransformation::getTestCaseName(testing::TestParamInfo<LayerTestsUtils::LayerTransformationParams> obj) {
+    InferenceEngine::Precision netPrecision;
+    InferenceEngine::SizeVector inputShapes;
+    std::string targetDevice;
+    InferenceEngine::details::LayerTransformation::Params params;
+    std::tie(netPrecision, inputShapes, targetDevice, params) = obj.param;
+
+    std::ostringstream result;
+    result << netPrecision.name() << "_" << targetDevice << "_" << toString(params);
+    return result.str();
+}
+
+void ConcatNeighboringGraphTransformation::SetUp() {
+    SetRefMode(LayerTestsUtils::RefMode::IE);
+
+    InferenceEngine::SizeVector inputShape;
+    InferenceEngine::Precision netPrecision;
+    InferenceEngine::details::LayerTransformation::Params params;
+    std::tie(netPrecision, inputShape, targetDevice, params) = this->GetParam();
+    auto ngPrecision = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+    const auto paramNode1 = std::make_shared<ngraph::opset1::Parameter>(ngPrecision, ngraph::Shape(inputShape));
+    const auto fakeQuantize1 = makeFakeQuantize(ngPrecision, paramNode1->output(0));
```

Comment:
ngraph builders contain its own makeFQ routines: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_functions/include/ngraph_functions/builders.hpp#L168. Is it possible to reuse?

Reply:
Reused, base test class was updated. thanks!